README.md

```diff
@@ -54,8 +49,8 @@ fn main() {
 }
 ```
 
-If you want to check for the program's output, you can use `print` or
-`print_exactly`:
+If you want to match the program's output _exactly_, you can use
+`prints_exactly`:
```

Comment:
> I tried hard to think of a different naming, where the strict variant would have the shorter name, but I couldn't find something nice.

Me neither! That's one of the main reason I made the exact method's name longer.

The previous version only had exact matching, and it was difficult to get right, because you had to include every new line (Maybe with carriage return on Windows? Who knows!) and control character.

This reminds of of another thing I want to support: Multiple fuzzy checks. I'll open an issue for that in a minute.